scripts/blender_load_replay.py

```diff
@@ -0,0 +1,348 @@
+                render_asset_map[instance_key] = obj
+
+                filepath = filepath.split("?")[0]
+                asset_info_by_key[instance_key] = asset_info_by_filepath[
+                    filepath
+                ]
+
+        if "stateUpdates" in keyframe:
+            for update_dict in keyframe["stateUpdates"]:
+                instance_key = update_dict["instanceKey"]
+                translation = update_dict["state"]["absTransform"][
+                    "translation"
+                ]
+                rotation = update_dict["state"]["absTransform"]["rotation"]
+                if instance_key not in render_asset_map:
+                    continue
+                obj = render_asset_map[instance_key]
+
+                obj.rotation_mode = "QUATERNION"
+
+                asset_info = asset_info_by_key[instance_key]
+
+                # note coordinate convention change for Blender
+                obj.location = (
+                    translation[0],
+                    -translation[2],
+                    translation[1],
+                )
+                obj.rotation_quaternion = (
+                    rotation[0],
+                    rotation[1],
+                    -rotation[3],
+                    rotation[2],
+                )
+
+                frame = asset_info["frame"]
+                if frame["up"] == [0.0, 1.0, 0.0]:
+                    pass
+                elif frame["up"] == [0.0, 0.0, 1.0]:
+                    obj.rotation_mode = "XYZ"
+                    obj.rotation_euler[0] -= 1.5708
+                else:
+                    raise NotImplementedError(
+                        "unexpected coordinate frame " + frame
+                    )
+
+        if do_add_anim_keyframes:
+            for instance_key in render_asset_map:
+                obj = render_asset_map[instance_key]
+                obj.keyframe_insert(data_path="location", frame=keyframe_index)
+                obj.keyframe_insert(
+                    data_path="rotation_quaternion", frame=keyframe_index
+                )
+
+    for o in bpy.context.scene.objects:
+        o.hide_set(False)
+    # To fix import issue where some robot models would have transparent links.
+    for m in bpy.data.materials:
+        m.show_transparent_back = False
+
+    add_lights = settings.get("lights", [])
+    for light in add_lights:
+        # Create light datablock
+        light_data = bpy.data.lights.new(
+            name=light["name"], type=light["type"]
+        )
+        light_data.energy = light["energy"]
+
+        # Create new object, pass the light data
+        light_object = bpy.data.objects.new(
+            name=light["name"], object_data=light_data
+        )
+
+        # Link object to collection in context
+        bpy.context.collection.objects.link(light_object)
+
+        # Change light position
+        light_object.location = light["location"]
+
+    print("")
+    if len(keyframes) > 1:
```

Comment:
We should update this message to include (1) if the ceiling or anything else was culled, and (2) what lights were spawned. This will help users troubleshoot when using this script.